CHANGELOG.md

```diff
@@ -21,6 +21,13 @@
 * `[jest-config]` Added restoreMocks config option.
   ([#5327](https://github.com/facebook/jest/pull/5327))
 
+## jest 22.1.1
+
+### Fixes
+
+* `[*]` Move from "process.exit" to "exit.
```

Comment:
`jest-cli`, `jest-runner` and `jest-runtime` all changed, so I went with `*`. Should I instead list all three?

Reply:
This is fine.